addons/account/views/report_invoice.xml

```diff
@@ -73,7 +73,7 @@
 
                                 <tr t-att-class="'bg-200 font-weight-bold o_line_section' if line.display_type == 'line_section' else 'font-italic o_line_note' if line.display_type == 'line_note' else ''">
                                     <t t-if="not line.display_type" name="account_invoice_line_accountable">
-                                        <td name="account_invoice_line_name"><span t-field="line.name" t-options="{'widget': 'text'}"/></td>
+                                        <td t-att-res-id="line.id" t-att-view-type="'form'" t-att-res-model="'account.move.line'"  name="account_invoice_line_name"><span t-field="line.name" t-options="{'widget': 'text'}"/></td>
```

Comment:
Just a change that allow trying the actions from client actions.

Reply:
fair enough, though the feature should be tested in its own right ; leaving this file alone for merging in master